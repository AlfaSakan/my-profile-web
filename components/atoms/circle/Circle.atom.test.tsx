import renderer from 'react-test-renderer';
import Circle from './Circle.atom';

describe('circle component test', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Circle>
          <p>test</p>
        </Circle>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
