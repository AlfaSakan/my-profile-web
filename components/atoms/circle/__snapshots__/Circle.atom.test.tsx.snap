// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`circle component test should render correctly 1`] = `
<div
  className="rounded-full bg-gray-300 mr-4 flex items-center justify-center overflow-hidden undefined"
>
  <p>
    test
  </p>
</div>
`;
